ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockCreator.java

```diff
@@ -208,8 +208,6 @@ protected Block createBlock(
       return new Block(blockHeader, new BlockBody(transactionResults.getTransactions(), ommers));
     } catch (final SecurityModuleException ex) {
       throw new IllegalStateException("Failed to create block signature", ex);
-    } catch (final CancellationException ex) {
-      throw new IllegalStateException("Attempt to create block was interrupted", ex);
```

Comment:
Won't this now be caught by the next `catch (Exception` block and wrapped in `IllegalStateException` anyway?